### ISSUE TITLE
Gating out events on axis edge

### ISSUE THREAD

**Issue by Biomiha** [DillonHammill/CytoRSuite#13]
Hi,

I've had some conversations with Mike Jiang about how best to gate out events that lie on axis edges (https://support.bioconductor.org/p/117610). Using the `mindensity` or `mindensity2` gates from `openCyto` generates a gate that is then plotted differently using ggcyto vs. cyto_plot (I'm a big fan BTW!). I've summarised the differences here:
http://rpubs.com/biomiha/470168.

I'm not sure what goes on in the background, i.e. initially I thought the mindensity output should set the gate dims to the left rather than the right of the gate but Mike was of the opinion that I should raise an issue here. https://support.bioconductor.org/p/117610/#118313

In summary, I'd like to capture events to the left of a 1D mindensity gate but cyto_plot plots the gate to the right.

Many thanks,
Miha

**Comment by gfinak** [comment]
Miha, I suspect what is happening is that the gate is treated as an
"exclusion" gate by openCyto. So cells not inside the gate are selected.
I'm not sure that should happen this way when the pop argument is "-", this
may be a bug in the mindensity2 gate as you pointed out in your first post.
We'll follow up on the openCyto end tomorrow.

Greg Finak

**Comment by DillonHammill** [comment]
Hi Biomiha,

Thanks for trying out the package! 

From the CytoRSuite side, plotting negated gates is not yet supported (I am working on this at the moment), the same goes for other boolean gates. Currently these types of gates should be skipped in cyto_plot but this is bypassed because you explicitly supplied the gate. When plotting a GatingSet there is no need to manually supply the gate. You can simply pass the name of the gate to the alias argument.

```{r}
cyto_plot(gs,
parent = "root",
alias = "nonDebris",
channels = c("FSC-A","SSC-A")
```

In the meantime to accomplish what you aim to acheive, I would use a manually drawn boundary gate. This will retain all events below the selected upper limit. This gate type is supported in 2 dimensions as well.

I will let you know when plotting boolean gates is supported.

Dillon

**Comment by Biomiha** [comment]
Thanks Dillon. I'll look forward to any updates.

**Comment by DillonHammill** [comment]
@biomiha this feature is now supported in CytoExploreR.

Simply set `negate = TRUE` with `cyto_gate_draw` or `cyto_plot`.
